packages/core/admin/server/content-types/User.js

```diff
@@ -89,7 +89,7 @@ module.exports = {
       configurable: false,
       private: true,
     },
-    preferedLanguage: {
+    preferredLanguage: {
```

Comment:
This will break users configuration when they update. we would have to provide an automatic column migration

Reply:
true. I was wondering how this is typically handled, assuming a Knex.js script/function is tucked in somewhere to search for the legacy spelling and rewrite under the new

Reply:
We would actually have to introduce something we have been planning but didn't need until know which is internal migration files to do that migration. I'll probably work on it and send you some info so you can add a dedicated migration for your needs :) Does that sound good ?

Reply:
That would be amazing! I'll keep an eye out, thank you